app/about_us/our_crew/page.js

```diff
@@ -0,0 +1,114 @@
+import styles from "./page.module.css";
+
+const OurCrew = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.ourCrew}>
+        <p>OUR CREW </p>
+        <h1>Who are we</h1>
+        <p>
+          Our crew is the heart and soul of Galactica. We are a diverse team of
+          seasoned space explorers, engineers, and visionaries who are united by
+          a common goal: to make space travel accessible and exciting for all.
+        </p>
+      </div>
+      <div className={styles.members}>
+        <section className={styles.memberSection}>
+          <img
+            src={`/crew/Sarah.png`}
+            alt="Sarah_Image"
+            width={300}
+            height={300}
+          />
+          <h3>Sarah Vega</h3>
```

Comment:
Great job, this should be an `h3` exactly! Only because we already have an `h1` and an `h2` for this section.